src/main/java/uk/gov/ida/eventemitter/LocalEncrypter.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.ida.eventemitter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Inject;
+import org.apache.commons.codec.binary.Base64;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+public class LocalEncrypter implements Encrypter {
+
+    private final String key;
+    private final String initVector;
+    private final ObjectMapper mapper;
+
+    @Inject
+    public LocalEncrypter(final String key,
```

Comment:
Couple of questions?
1) Why `Local`?
2) Until we are ready to use KMS, how do we ensure that all of the Hub components (and the lambda) use the same key? Are we just going to stick a hard coded key in the config?